src/assegai/Autoloader.php

```diff
@@ -123,8 +123,16 @@ private function getLegacy($classname)
                         'Exception' => 'exceptions',
                         'Model' => 'models',
                         'View' => 'views');
-                    $filename = $this->conf->get('apps_path') . '/' . strtolower($app) . '/'
-                        . $paths[$type] . '/' . str_replace('_', '/', strtolower($class)) . '.php';
+                    if(array_key_exists($type, $paths))
+                    {
+                        $filename = $this->conf->get('apps_path') . '/' . strtolower($app) . '/'
+                            . $paths[$type] . '/' . str_replace('_', '/', strtolower($class)) . '.php';
+                    }
+                    else
+                    {
+                        $filename = false;
```

Comment:
Set `filename = "";` here, instead of false. Later on, I test for `filename != ""`, so for consistency's sake it's better.